tokenizers/src/utils.rs

```diff
@@ -93,8 +98,11 @@ pub fn truncate_encodings(
                 Err(Box::new(Error::SecondSequenceNotProvided))
             }?;
 
-            if target.get_ids().len() > params.max_length {
-                target.truncate(params.max_length, params.stride);
+            let target_len = target.get_ids().len();
+            if target_len > to_remove {
+                target.truncate(target_len - to_remove, params.stride);
+            } else {
+                return Err(Box::new(Error::SequenceShorterThanNumberTokensToRemove));
```

Comment:
What do you think about `SequenceTooShort`? I would also change the message in the `Display` implementation to say something like `Sequence to truncate too short to respect the provided max_length`.

Reply:
Oh yeah sure, I really thought this name was too long but couldn't find anything better ^^' 👍

Reply:
was the name of the option longer than the number of tokens to remove? 😅